core/shared/src/main/scala/org/threeten/bp/Duration.scala

```diff
@@ -89,7 +89,7 @@ object Duration {
    * @throws ArithmeticException
    *   if the input days exceeds the capacity of { @code Duration}
    */
-  def ofDays(days: Long): Duration = create(Math.multiplyExact(days, 86400), 0)
+  def ofDays(days: Long): Duration = create(Math.multiplyExact(days, 86400.toLong), 0)
```

Comment:
Having to add `.toLong` at so many places is an unfortunate hack... Shouldn't we just add these overloads to Scala Native upstream?
/cc @lolgab

Reply:
I don't know if this is a problem with Scala Native. Probably the automatic conversion from `Int` to `Long` is deprecated if you upgraded Scala? I don't know,  in the Scala Library there are two overloads [with two Longs](https://www.scala-lang.org/api/current/scala/math/index.html#multiplyExact(x:Long,y:Long):Long) and [with to Ints](https://www.scala-lang.org/api/current/scala/math/index.html#multiplyExact(x:Long,y:Long):Long).
Anyway, you can use the literal instead of `.toLong` since the number is a fixed one:
```suggestion
  def ofDays(days: Long): Duration = create(Math.multiplyExact(days, 86400L), 0)
```

Reply:
I'm actually not upgrading any Scala version. It is a bit of a mystery to me how it passed CI before.

But thanks for the literal suggestion, much cleaner that way :+1: